main.js

```diff
@@ -456,14 +502,21 @@ function startAliveInterval() {
     config.system = config.system || {};
     config.system.statisticsInterval = parseInt(config.system.statisticsInterval, 10) || 15000;
     config.system.checkDiskInterval  = (config.system.checkDiskInterval !== 0) ? parseInt(config.system.checkDiskInterval, 10) || 300000 : 0;
+    if (!compactGroupController) {
```

Comment:
Why do we need this state? --> Comment ;)

Reply:
because setting can bié different per host (in multihost env) and status is currently only in "core config json" ... so having it visible as state makes sense for me, also to show in Admin or such